distro/runtime/concierge/smarthome.xargs

```diff
@@ -3,7 +3,7 @@
 # Version configuration
 -Dconcierge.version=5.0.0
 -Djetty.version=9.3.10
--Desh.version=0.9.0.b1
+-Desh.version=0.9.0-SNAPSHOT
```

Comment:
I moved over to Snapshot as audio and voice support have not been part of b1 release.
I can go back to b1 but have to disable audio/voice then again
Whats your recommendation?